add `kvm-ok` diagnostics suggestion (#14912)
docker/docker.github.io: desktop/linux/install.md

$ modprobe kvm_amd    # AMD processors
```

> If the above commands fail, diagnostic info can be surfaced by running:

Reviewer: ```suggestion
If the above commands fail, you can view the diagnostics by running:
```
